deployments/cloudfoundry/buildpack/README.md

```diff
@@ -0,0 +1,168 @@
+# Note: This will provide the Collector for the app, but the Collector will not be running.
+$ cf push <app-name> -b otel_collector_buildpack -b <main_buildpack>
+
+# How to run an application with a manifest.yml file:
+# Option 1)
+$ cf push
+# Option 2)
+$ cf push <app-name> -b otel_collector_buildpack -b <main_buildpack> -f manifest.yml
+```
+
+Note: This buildpack requires another buildpack to be supplied after it, it is not allowed to
+be the last one for an app. Also, the manifest.yml file will need to provide the
+command to run the OpenTelemetry Collector as a sidecar for the application.
+
+## Configuration
+
+The following only applies if you are using the `otelconfig.yaml` config
+provided by the buildpack.  If you provide a custom configuration file for the OpenTelemetry Collector 
+in your application (and refer to it in the sidecar configuration), these might not
+work unless you have preserved the references to the environment variables in the config file.
+For proper functionality, the `OTEL_CONFIG` environment variable must point to
+the configuration file, whether using the default or a customer version.
+
+Set the following environment variables with `cf set-env` as applicable to configure this buildpack, or
+include them in the `manifest.yml` file, as shown in the [included example](#sidecar-configuration).
+
+Required:
+- `RLP_GATEWAY_ENDPOINT` - The URL of the RLP gateway that acts as the proxy for the firehose,
+    e.g. `https://log-stream.sys.<TAS environment name>.cf-app.com`
+- `UAA_ENDPOINT` - The URL of UAA provider,
+    e.g. `https://uaa.sys.<TAS environment name>.cf-app.com`
+- `UAA_USERNAME` - Name of the UAA user.
+- `UAA_PASSWORD` - Password for the UAA user.
+- `SPLUNK_ACCESS_TOKEN` - Your Splunk organization access token.
+- The Splunk Observability Suite requires an endpoint, so one of the two options below must be specified:
+    1) - `SPLUNK_INGEST_URL` - The ingest base URL for Splunk. This option takes precedence over SPLUNK_REALM.
+       - `SPLUNK_API_URL` - The API server base URL for Splunk. This option takes precedence over SPLUNK_REALM.
+    2) - `SPLUNK_REALM` - The Splunk realm in which your organization resides. Used to derive SPLUNK_INGEST_URL
+       and SPLUNK_API_URL.
+
+Optional:
+- `OS` - Operating system that Cloud Foundry is running. Must match format of Otel Collector executable name.
+    Default: `linux_amd64`
+- `OTEL_CONFIG` - Local name of OpenTelemetry config file. Default: `otelconfig.yaml`
+- `OTEL_VERSION` - Executable version of OpenTelemetry Collector (contrib) to use. The buildpack depends on features present in version
+    v0.48.0+. Default: `latest`. Example valid value: `v0.48.0`.
+    Note that if left the default value, the latest version will be found and later variable references will be
+    to a valid version number, not simply the word "latest".
+- `OTEL_BINARY` - OpenTelemetry Collector executable file name. Default: `otelcontribcol_$OS-v$OTEL_VERSION`
```

Comment:
I think we should remove references to `contrib`.
```suggestion
- `OTEL_BINARY` - OpenTelemetry Collector executable file name. Default: `otelcol_$OS-v$OTEL_VERSION`
```

Reply:
I've removed contrib references.